modin/engines/base/frame/partition_manager.py

```diff
@@ -569,16 +571,53 @@ def to_numpy(cls, partitions, **kwargs):
     @classmethod
     def from_pandas(cls, df, return_dims=False):
         """Return the partitions from Pandas DataFrame."""
+
+        def update_bar(pbar, f):
+            if ProgressBar.get():
+                pbar.update(1)
+            return f
+
         num_splits = NPartitions.get()
         put_func = cls._partition_class.put
         row_chunksize, col_chunksize = compute_chunksize(df, num_splits)
+
+        bar_format = (
+            "{l_bar}{bar}{r_bar}"
+            if os.environ.get("DEBUG_PROGRESS_BAR", "False") == "True"
```

Comment:
Where is `DEBUG_PROGRESS_BAR` defined? It should probably be `MODIN_PROGRESS_BAR` that is introduced in `modin.config`.

Reply:
It's used to make sure that the output of the bar is correct during visual inspection. It's not meant for end user use.

Reply:
Was it documented somewhere? Or we should remember it?

Reply:
It was not documented, since the progress bar is still early. It should go away after full development is complete, but the progressbar will touch a large amount of code so we want to do it in small pieces we can verify.

Reply:
Okay, I see.